README.md

```diff
@@ -10,19 +10,35 @@ At present `stellar-core-backup` makes a few assumptions about the environment t
 
 AWS credentials can be exported as environment variables or permissions can be granted by an IAM instance role.
 
+## GPG Backup Signing and Verification
+
+GPG verificaiton is enabled by default. Unless the user disables this securtiy feature with the --no-verify option, GPG signing and verification checks will be run.
+
+The appropriate GPG signing keys need to be in place and the key_id selection is configured in the configuration file.
+
+To install the public Stellar backup key for verification of Stellar provided backups, used the --getkey flag and the provided key-ID contained in the config/sample.yaml file. Change this value for your own key.
+
+`stellar-core-backup --getkey`
+
+The keyservers used to serve this key from belong to the SKS Keyserver service cluster hkp://pool.sks-keyservers.net. 
+
+If you try to verify a backup that has not been signed, you will generate an error.
```

Comment:
Not sure we need this information as all the backups will eventually be signed by SDF